lib/IDE/Utils.cpp

```diff
@@ -875,11 +879,15 @@ Expr *swift::ide::getBase(ArrayRef<Expr *> ExprStack) {
   Expr *ParentE = getContainingExpr(ExprStack, 1);
   if (ParentE && isa<FunctionConversionExpr>(ParentE)) {
     ParentE = getContainingExpr(ExprStack, 2);
+  } else if (ParentE && isa<MetatypeConversionExpr>(ParentE)) {
+    ParentE = getContainingExpr(ExprStack, 2);
   }
```

Comment:
I would merge these branches and just check for `ImplicitConversionExpr` instead. For completeness, we _probably_ ought to iterate up the stack if we encounter multiple `ImplicitConversionExpr`s, though I can't immediately think of a case where that happens, so I don't think it's crucial.

Reply:
this one was removed as well